app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
You should follow one code style for your project, place `self` on a new line, and write `__init__` as in `CarWashStation`.